_benchmark/workloads/index.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: default
+title: Workload reference
+nav_order: 60
+has_children: true
+---
+
+# OpenSearch Benchmark workload reference
+
+A workload is a specification of one or more benchmarking scenarios. A workload typically includes the following:
+
+- One or more data streams that are ingested into indices.
+- A set of queries and operations that are invoked as part of the benchmark.
+
+## Anatomy of a workload
+
+The following example workload shows all of the essential elements need to create a workload.json file. You can run this workload in your own benchmark configuration, in order to understand how all element coalesce together:
+
+```json
+{
+  "description": "Tutorial benchmark for OpenSearch Benchmark",
+  "indices": [
+    {
+      "name": "movies",
+      "body": "index.json"
+    }
+  ],
+  "corpora": [
+    {
+      "name": "movies",
+      "documents": [
+        {
+          "source-file": "movies-documents.json",
+          "document-count": 11658903, # Fetch document count from command line
+          "uncompressed-bytes": 1544799789 # Fetch uncompressed bytes from command line
+        }
+      ]
+    }
+  ],
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "create-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "cluster-health",
+        "request-params": {
+          "wait_for_status": "green"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "bulk",
+        "bulk-size": 5000
+      },
+      "warmup-time-period": 120,
+      "clients": 8
+    },
+    {
+      "operation": {
+        "name": "query-match-all",
+        "operation-type": "search",
+        "body": {
+          "query": {
+            "match_all": {}
+          }
+        }
+      },
+      "iterations": 1000,
+      "target-throughput": 100
+    }
+  ]
+}
+```
+
+A workload usually consists of the following elements:
+
+- [indices]({{site.url}}{{site.baseurl}}/benchmark/workloads/indices/): Defines the relevant indices and index templates used for the workload.
+- [corpora]({{site.url}}{{site.baseurl}}/benchmark/workloads/corpora/): Defines all documents corpora used for the workload.
+- `schedule`: Defines in what order the workload performs an action. It can use either the `operations` element to define the operations or place the operations in line.
+- `operations`: **Optional**. Describes which operations are available for this track and how they are parametrized. 
+
+### Indices
+
+To create an index, you need to specify the `name` of the index. If you want to add definitions for your index, use the `body` option and point it to the JSON file containing the index definitions.
+
```

Comment:
Should we also have a quick link at the end of the each summary section in the index.md where the user can click and learn about more in-depth information? Example:
```
To learn more about indices, visit [indices page]({{site.url}}{{site.baseurl}}/benchmark/workloads/indices/).
```